pkg/kubelet/qos/doc.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// package qos contains functions that support quality of service in Kubernetes.
+// The Kubelet supports three tiers of pods. Guaranteed pods  will receive the highest priority
```

Comment:
Done, and corrected some errors (the doc suggests that QoS is per pod, but it's per-resource, per-container)